tests/Unit/Domain/Regulation/MeasureTest.php

```diff
@@ -35,8 +36,9 @@ public function testMeasure(): void
         $this->assertSame($date, $measure->getCreatedAt());
```

Comment:
(Détail) Vérifier que le maxSpeed est bien mis à null ?

```suggestion
        $this->assertSame($date, $measure->getCreatedAt());
        $this->assertNull($measure->getMaxSpeed());
```